src/tests/ftest/container/full_pool_container_create.yaml

```diff
@@ -17,3 +17,10 @@ pool:
 
 container:
   control_method: daos
+  oclass: OC_SX
+  type: POSIX
+  object_qty: 1
+  record_qty: 1
+  data_size: 1048576
+  akey_size: 5
+  dkey_size: 5
```

Comment:
Where are these used by fill_container?

Reply:
These are used by the new `TestContainer.fill()` method:
- `data_size` sets the initial amount of data to write in the inner while loop that will get halved when we hit DER_NOSPACE until its 0
- 'object_qty', 'record_qty', `akey_size`, and `dkey_size` are passed by `self.write_objects()` into `TestrContainerData.write_object()`